code/modules/cargo/packs.dm

```diff
@@ -307,7 +307,7 @@
 					/obj/item/clothing/suit/armor/vest)
 	crate_name = "armor crate" */
 
-
+/* //SKYRAT EDIT START - CARGO ARMAMENTS
 /datum/supply_pack/security/disabler
```

Comment:
We can't order replacement disablers as security anymore with our departmental order console. In fact, this entire PR completely screws over Security's departmental order console. We can't even buy shit with our budget anymore since we can't buy crates as far as I can tell.

Reply:
figured out how to buy crates w/ the budget but the Departmental Order Console issue still stands

Reply:
Disabler crates are back, and will touch up on consoles later, possibly